packages/tools/pactjs-cli/src/utils/callLocal.ts

```diff
@@ -0,0 +1,40 @@
+import fetch from 'cross-fetch';
+
+export async function callLocal(
+  apiHost: string,
+  body: string,
+): Promise<{
+  textResponse: string | undefined;
+  jsonResponse:
+    | {
```

Comment:
What does the leading pipe `|` do here? I'm not familiar with this syntax

Reply:
No idea either, but Prettier won't let me remove it.